### ISSUE TITLE
Parallel data

### ISSUE THREAD

**Issue by fginter** [ufal/conll2017#11]
I'll open an issue for this. Our suggestion here in Turku is to provide the participants with parallel data and/or dictionaries and/or multilingual embeddings to support the development of lexicalized transfer methods in the Shared Task. Our argument is that if we do not allow any such resources, we limit people to delexicalized methods which is 1) not a realistic setting and 2) does not use UD to its full potential.

Opinions?

**Comment by dan-zeman** [comment]
Fine with me, but we need to know (soon, i.e. within 2 weeks) how much of this we are able to provide ourselves, and how much is just left open for the case that someone has it, wants to use it and is willing to share it.

I am not against parallel data but I do not want to commit to collecting them. We could take the easy way and say that Europarl & JRC Acquis are allowed, but there are non-European languages such as Persian and Hindi, for which parallel data exist but are not covered by those resources.

**Comment by fginter** [comment]
Maybe we could let them use [OPUS](http://opus.lingfil.uu.se/) and wiktionary? [edit: link to OPUS]

**Comment by fginter** [comment]
One extra massively parallel corpus is the WatchTower corpus from Copenhagen. They do release it, and I think it's 300+ languages. Agreed with @dan-zeman that we do not want to hunt down individual language pairs - the logistics would be a challenge. So OPUS + maybe WatchTower would be pretty much off-the-shelf resources we can simply allow. Especially OPUS is pretty much a standard, very widely used resource.

**Comment by foxik** [comment]
I think that providing existing corpora is fine, so using OPUS (which includes EuroParl, JRC, and others) seems a good idea. I could not find the WatchTower one.

As for the dictionaries, I think some multilingual dictionary is planned (but I do not know which one).

As for multilingual embeddings -- it could be useful for the participants if we provide some "basic" multilingual embeddings. We can do it similarly to the word embeddings -- we could provide some multilingual embeddings computed only from the data available to the participants -- that way, the participants can compute custom multilingual embeddings without any loss (compared to the situation when the provided embeddings would be computed on larger data not available as resources for the participants). However, do you have any method in mind which we would use to compute the multilingual embeddings? To my knowledge, there is not "a universal method" (compared for example to skip-gram). Also, we probably will not have parallel data for all languages -- so maybe we should compute the multilingual embeddings only using word embeddings + projections computed from parallel dictionary...

**Comment by fginter** [comment]
- WatchTower is not yet downloadable, but I'm sure we can get it if we want it.
- Yes, multilingual embeddings can be pre-trained. Of course we need to be careful not to overcommit ourselves.
- We definitely should only pre-train anything on the data we otherwise distribute and publish upfront how the pre-training was done

**Comment by martinpopel** [comment]
> do you have any method in mind which we would use to compute the multilingual embeddings? 

What about trans-grams?
http://www.aclweb.org/anthology/D15-1131.pdf
I think it's better than BilBOWA and better than projecting existing monolingual embeddings.
(See also [another version of the paper](https://arxiv.org/abs/1601.02502) and [an overview](http://www.aifb.kit.edu/images/b/b4/NAACL-HLT-2016-Camera-Ready.pdf))

**Comment by dan-zeman** [comment]
Note that OPUS contains EuroParl but it does not contain JRC-Aquis. No big deal I think, as there are roughly the same 21-22 languages. Bible and/or Watchtower would be more interesting because of their set of languages.

OK, so the minimum we can say (currently in my new draft of the proposal):

_To support multi-lingual and cross-lingual approaches and model transfers, participants are allowed to use data from the OPUS parallel corpus (http://opus.lingfil.uu.se/). We will not redistribute this data; participants are simply referred to the OPUS website._

If anyone wants to commit to preparing and providing data beyond this, that is fine. It is probably fine even if it is not explicitly mentioned in the task proposal. However, if you feel it should be mentioned there, please let me know.

**Comment by fginter** [comment]
At least for Finnish-English, JRC-Aquis is in OPUS. Maybe it's there for some language pairs and not for others...? I now have a local copy of WatchTower. It's about 40-50K sentences for each language pair, so it's not that big. Copenhagen plans to contribute it to OPUS, so it will likely be there when the shared task happens. I think your proposed phrasing is perfect.  Thanks!

**Comment by dan-zeman** [comment]
Hmm I was mislead by seeing JRC-Acquis among "links to other resources", and not seeing it next to Europarl. I just have been blind, it is in the list.
